examples/multipart_form.rs

```diff
@@ -12,7 +12,13 @@ use std::net::SocketAddr;
 
 #[tokio::main]
 async fn main() {
-    tracing_subscriber::fmt::init();
+    // Set the RUST_LOG, if it hasn't been explicitly defined
+    if std::env::var("RUST_LOG").is_err() {
+        std::env::set_var("RUST_LOG", "multipart_form=debug")
```

Comment:
```suggestion
        std::env::set_var("RUST_LOG", "multipart_form=debug,tower_http=debug")
```

Since the example is using `TraceLayer`

Reply:
Oh I missed that this had any tower_http wiring, thanks!